src/phpDocumentor/Plugin/Graphs/ServiceProvider.php

```diff
@@ -11,22 +11,28 @@
 
 namespace phpDocumentor\Plugin\Graphs;
 
-use Cilex\Application;
-use Cilex\ServiceProviderInterface;
 use phpDocumentor\Plugin\Graphs\Writer\Graph;
 use phpDocumentor\Transformer\Writer\Collection;
 
-class ServiceProvider implements ServiceProviderInterface
+class ServiceProvider
```

Comment:
should be final.
And it looks like this is an addition on the core/serviceprovider? There must be a beter name for those classes, since they are configuring the Writer\Collection?